.github/dependabot.yml

```diff
@@ -118,3 +136,30 @@ updates:
     schedule:
       interval: weekly
       day: sunday
+  - package-ecosystem: gomod
+    directory: /launcher
```

Comment:
These don't seem right. Is the `launcher` directory local?

Reply:
Ah yeah it is, but it's [.gitignore'd ](https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/2ac28454bfd38c4aff508c67bf22a66cbedaf1da/.gitignore#L27). I think that's from earlier code when we used a launcher. Will fix